index.js

```diff
@@ -103,7 +103,7 @@ class ShopifyToken {
   /**
    * Verify the hmac returned by Shopify.
    *
-   * @param {Object} query The query string object
+   * @param {Object} query The query object, e.g. `{hmac: "fd4c", timestamp: 1337178173}`
```

Comment:
```suggestion
   * @param {Object} query The parsed query string
```

Reply:
I think that makes it sound like it expects a string as input

Reply:
Type is specified. `Object`.

Reply:
It's not clear that is doesn't expect a `String` object though, like `String("hmac=fd4c&timestamp=1337178173")`

Reply:
The same applies to the ctor options object then. It does not make much sense. It type was a string it would have been `@param {String}` like other string parameters.

Reply:
ok

Reply:
Also there is an example in the [README](https://github.com/lpinca/shopify-token#shopifytokenverifyhmacquery) that make it very clear in my opinion.